.github/workflows/tests.yaml

```diff
@@ -21,13 +21,18 @@ jobs:
         php-version: [ '8.1', '8.2', '8.3' ]
         dependencies: [ locked, lowest, highest ]
         coverage-driver: [ pcov, xdebug ]
-        symfony-version: [ '5.4.*', '6.*.*' ]
+        symfony-version: [ '5.4.*', '6.*.*', '7.*.*' ]
         include:
           - operating-system: windows-latest
             php-version: '8.1'
             dependencies: locked
             coverage-driver: xdebug
             symfony-version: '5.4.*'
+        exclude:
+          - php-version: '8.1'
+            symfony-version: '7.*.*'
+          - dependencies: 'locked'
```

Comment:
it seems like we could exclude symfony-version = 6 with locked dependecies either as it doesn't make sense as we locked with symfony 5.4 components so far.